packages/feeds-client/src/feed/feed.ts

```diff
@@ -229,6 +243,10 @@ export class Feed extends FeedApi {
     return this.state.getLatestValue();
   }
 
+  hasActivity(activity_id: string) {
```

Comment:
```suggestion
  hasActivity(activityId: string) {
```

Reply:
Hmm, okay - I probably misunderstood in that case; I'll add this too